.github/CODEOWNERS

```
@@ -25,6 +25,7 @@
 /bonsai/                                 dev@onemorecloud.com
 /botprise/                               @codechefnisha-bp nisha.kumari@botprise.com
 /buddy/                                  support@buddy.works
+/buoyant_cloud/                          @BuoyantIO/buoys
```

Could you please add `makretplace-review as a CODEOWNER:
```suggestion
/buoyant_cloud/                          @BuoyantIO/buoys @DataDog/marketplace-review
```